imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc

```diff
@@ -0,0 +1,157 @@
+---
+title: Immersve Conducted KYC
+slug: guides/immersve-conducted-kyc
+---
+
+In the Immersve conducted KYC mode, Immersve facilitates the entire KYC process
```

Comment:
```suggestion
In the Immersve conducted [KYC mode](/guides/kyc-modes), Immersve facilitates the entire KYC process
```

Reply:
but wait, this page is about this mode itself, why would we link to the intro page here?